src/place-screen/hooks/use-departures-data.ts

```diff
@@ -337,6 +338,11 @@ export function useDeparturesData(
     !!state.tick,
   );
 
+  useEffect(() => {
+    const estimatedCalls = state.data;
+    estimatedCalls && potentiallyMigrateFavoriteDepartures(estimatedCalls);
+  }, [state.data, potentiallyMigrateFavoriteDepartures]);
```

Comment:
Is `potentiallyMigrateFavoriteDepartures` a stable reference?

Reply:
Good question!
Yes, both code walkthrough and testing indicates that it is stable.

Reply:
Aha 😅
I just looked at the current implementation and whether there was a way to trigger repeated potentiallyMigrateFavoriteDepartures changes from using the app. Will look into safeguarding it for the future as well :+1: